test/plugins/builds.test.js

```diff
@@ -2264,12 +2264,20 @@ describe('build plugin test', () => {
                     }]);
 
                     return newServer.inject(options).then(() => {
-                        jobCconfig.start = false;
-                        jobCconfig.parentBuilds = { 123: {
-                            jobs: { a: 12345, b: null },
-                            eventId: '8888'
-                        } };
-                        assert.calledWith(buildFactoryMock.create, jobCconfig);
+                        assert.calledWith(buildFactoryMock.create.firstCall, {
```

Comment:
does this cover all scenarios ? don't we need more test cases ? Seeing that coverage has decreased.

Reply:
added a test